search/src/Components/wrapper/upload.md

```diff
@@ -20,6 +27,7 @@ users to freely share and adapt the material for any purpose, even
 commercially, given appropriate attribution and that there are no additional
 restrictions made.
 
```

Comment:
Should add something along the lines of (taken from creative commons website): "This is a human-readable summary of (and not a substitute for) the license".